scripts/schedule-webworker.js

```diff
@@ -0,0 +1,122 @@
+function getEndTime(date, startTime, endTime, totalNumber, number) {
+  const timezone = new Date().toString().match(/([A-Z]+[\+-][0-9]+.*)/)[1];
+  const timeStart = new Date(date + ' ' + startTime + ' ' + timezone).getTime();
+  const timeEnd = new Date(date + ' ' + endTime + ' ' + timezone).getTime();
+  const difference = Math.floor((timeEnd - timeStart) / totalNumber);
+  const result = new Date(timeStart + difference * number);
+  return result.getHours() + ':' + result.getMinutes();
+}
+
+function getDuration(date, startTime, endTime) {
+  const timezone = new Date().toString().match(/([A-Z]+[\+-][0-9]+.*)/)[1];
+  const timeStart = new Date(date + ' ' + startTime + ' ' + timezone).getTime();
+  const timeEnd = new Date(date + ' ' + endTime + ' ' + timezone).getTime();
```

Comment:
timezone, timeStart and timeEnd are also calculated infunction above. Better to move this into common function